test/reducers/dist.js

```diff
@@ -233,29 +204,21 @@ test('reducers/dist/dist_s3access', (t) => {
     }
   };
   const newState = reducer(initialState, action);
-  t.deepEqual(expectedState, newState);
+  t.deepEqual(expectedState.s3Access, newState.s3Access);
 });
 
 test('reducers/dist/dist_s3access_inflight', (t) => {
   const initialState = {
-    apiLambda: {
-      inflight: false
-    }
-  };
-  const action = {
-    type: DIST_S3ACCESS_INFLIGHT
-  };
-
-  const expectedState = {
     apiLambda: {
       inflight: false
     },
-    s3Access: {
-      inflight: true
-    }
+    s3Access: {}
   };
+  const action = { type: DIST_S3ACCESS_INFLIGHT };
   const newState = reducer(initialState, action);
-  t.deepEqual(expectedState, newState);
+
+  t.false(newState.apiLambda.inflight);
+  t.true(newState.s3Access.inflight);
```

Comment:
I think I still prefer the explicit expected state to compare like in the old test, but again. you don't have to change it.